recipes/crabs/meta.yaml

```diff
@@ -33,8 +36,8 @@ requirements:
     - numpy
     - pandas >=0.23.4
     - matplotlib-base
-    - wget
-    - muscle >=3.8.31
+    - wget =1.21.3
```

Comment:
Why exactly 1.21.3 ?

Reply:
not sure - just copied that from the other PR. I think we can go for the latest version here.

Reply:
I guess any new semver compatible version should be OK.
Pinning with exact versions is usually not recommended.